test/mocks/FutarchyChallengeFactoryMock.sol

```diff
@@ -2,7 +2,7 @@ pragma solidity ^0.4.24;
 
 import "./FutarchyChallengeMock.sol";
 
-contract FutarchyCHallengeFactoryMock {
+contract FutarchyChallengeFactoryMock {
```

Comment:
That H is evil...

Reply:
or....did I............push broken tests?? Impossible...

Reply:
Haha files on Windows are case insensitive, so that might be it? It's one of those pesky OS differences...